dev-util/tup/tup-0.7.10.ebuild

```diff
@@ -0,0 +1,103 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit linux-info toolchain-funcs
+
+DESCRIPTION="A file-based build system"
+HOMEPAGE="http://gittup.org/tup"
+# Tup itself is GPLv2, but it bundles differently licensed software:
+# - lua: MIT
+# - sqlite (unused in this ebuild): public domain
+# - inih: 3-clause BSD
+# - red-black tree implementation: 2-clause BSD
+# - queue implementation: 3-clause BSD
+LICENSE="GPL-2 MIT public-domain BSD BSD-2"
+SLOT="0"
+
+if [[ "${PV}" == 9999 ]]; then
+	EGIT_REPO_URI="https://github.com/gittup/tup.git"
+	inherit git-r3
+else
+	SRC_URI="https://github.com/gittup/tup/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+
+DEPEND="
+	dev-db/sqlite:=
+	dev-libs/libpcre:=
+	sys-fs/fuse:=
```

Comment:
Does this work with both fuse's slots? You may need to restritct it with `:0=` or `:3=`.

Reply:
You're right, it needs to be restricted to `:3=`